### DISCUSSION TITLE
Flow past cylinder - Lift coefficient

### DISCUSSION THREAD

**Discussion by lauguimel** [fppimenta/rheoTool#61] [General]
Dear Fppimenta
I am interested in calculating the lift coefficient of a cylinder in a channel with a viscoelastic flow, depending on its y-position. I made some computations and got some results and to explain the results obtained, I then checked the Newtonian case for Re=0 (ok, I should have done this in first place !).
Unexpectedly, the lift coefficient for the Newtonian case doesn't remain null when moving the cylinder in the y direction. Even if at the center it is null ($\approx=10^{-14}$), it goes up to $\approx=10^{-4}$, in the upper half part of the channel which is very similar to the results of the elastic case. Based on literature, this lift coefficient should remain null for a Stokes flow wherever the position in the y-axis.
I have doubled checked that the convective term is disabled in fvSchemes, tried multiple convergence test (in time, mesh, etc) and I am not able to figure out what is wrong. 

Have you an idea ?
If you want to have a look, I set up a minimum working example here based on your tutorial:
https://cloud.univ-grenoble-alpes.fr/index.php/s/drbj62AKBwZMNBx

Thanks a lot in advance for any suggestion you may have !
Guillaume

**Comment by fppimenta** [comment]
It's not an issue, but a question related with general fluid mechanics. Please do not use this space for such purpose (there are certainly better places).

Unless I am missing something, what you are getting is the expected behavior. Why should Cl = 0 for an offset cylinder in a Poiseuille flow? It can be << Cd, but not null, as there is loss of symmetry about the transverse direction.

**Reply by lauguimel** [reply]
Hello Francisco
Thank you for transferring the question to discussion, indeed this is not an issue. 
Many papers state that the lift coefficient scale as 0(Re^2), so it should vanish in my case. I also ran the same simulation of Stokes flow with Comsol and the lift coefficient remains close to zero even when the cylinder is not located at the center. 
For Navier-Stokes equation, there is no discussion, there is a lift acting but for Stokes flow I am still confused about the results obtained with openFoam...I thus suspected a mistake in my configuration file.

I'll keep digging !
Thank you for your quick answer!

**Comment by fppimenta** [comment]
Can you share a paper showing 'the expected results'?

**Reply by lauguimel** [reply]
In [Bretherton 1962](https://www.cambridge.org/core/journals/journal-of-fluid-mechanics/article/abs/motion-of-rigid-particles-in-a-shear-flow-at-low-reynolds-number/72E54E271F01F5F5AF23178E64DCD302) it is theoretically shown that there is no lateral force in Stokes flow. It is also mentioned in the introduction of [Ho 1974](https://www.cambridge.org/core/journals/journal-of-fluid-mechanics/article/abs/inertial-migration-of-rigid-spheres-in-twodimensional-unidirectional-flows/E4BDB1746BD7EC37C0FAABF52EC19147) or in the book of [Morris](https://www.cambridge.org/core/books/physical-introduction-to-suspension-dynamics/B8EB2B34C3D3893D7E22884710AA7E7A): _We saw in Chapters 1 and 2 that a neutrally buoyant spherical particle at zero Reynolds number will follow the motion of the fluid except for a lag owing to the “curvature” of the flow profile associated with the quadratic term (r/R)². Hence, at Re=0 the particle will not move laterally toward either the wall or the center-line – to do so would violate the reversibility property of Stokes flow._ (pp201-202)
